src/replacements/gcn-circular/index.ts

```diff
@@ -5,7 +5,7 @@ function normalize(match: string) {
   return parseFloat(match.replace(/a$/i, '.5'))
 }
 
-const preamble = /GCN(?:s|\s+Circulars?|\s+Circs?\.)?\s*/.source
+const preamble = /\(?GCN(?:s|\s+Circulars?|\s+Circs?\.?)?\s*(?:#)?/gi.source
```

Comment:
this is case insensitive now too, right? We should also add a test for a lowercase reference

Reply:
Yes.